src/main/java/com/partymusicapp/models/Follow.java

```diff
@@ -14,11 +14,14 @@ public class Follow {
 
     @ManyToOne
     @JoinColumn
-    private Follow followerID;
+    private User follower;
 
     @ManyToOne
     @JoinColumn
-    private Follow followedID;
+    private User followed;
+
+    @Column
+    private int noOfFollowers;
```

Comment:
In Follow nu ai nevoie de noOfFollowers. Asta ai in User. Ideea ii ca fie care obiect Follow ii special pentru fiecare follow de la follower la followed. Aici va fi tot timpul 1 dupa implementarea ta. Am schimbat in Follow Service sa dea update la user